h/api/search/query.py

```diff
@@ -56,16 +56,15 @@ def build(request_params, userid=None, search_normalized_uris=False):
     if uri_param is None:
         pass
     elif search_normalized_uris:
-        filters.append(_term_clause_for_uri(uri_param))
+        filters.append(_filter_for_uri_term(uri_param))
     else:
-        matches.append(_match_clause_for_uri(uri_param))
+        filters.append(_filter_for_uri_match(uri_param))
```

Comment:
Oh I see, you needed to make this change to the old URI matching in order to enable the change (in the second commit) to combining multiple search arguments with should instead of must? That makes sense